common/shlibs

```diff
@@ -3413,3 +3413,5 @@ libfrr.so.0 libfrr-6.0_1
 libfrrospfapiclient.so.0 libfrrospfapiclient-6.0_1
 liborocos-kdl.so.1.4 orocos-kdl-1.4.0_1
 libibumad.so.3 rdma-core-22.1_1
+libhtp.so.2 suricata-4.1.2_2
+
```

Comment:
extra blank line, remove

Reply:
this is not resolved

Reply:
wait. when commiting the rocksdb package you wanted an extra \n at the end of the file. so this changed? no \n after suricata-4.1.2_2 ?